x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/types.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Meta } from '../../../../../common/types';
+
+export interface Curation {
+  id: string;
+  last_updated: string;
+  queries: string[];
```

Comment:
I'm guessing we'll fill in these types a bit more in the future to be more specific than `object[]`?

Reply:
I don't think we do, I just copied this over from ent-search. Do you want me to just for specificity? To be honest I'm not 100% sure we use this much if at all (possibly will know for sure when I reach porting over curation pages)

Reply:
After looking a bit more ahead I'm 75% confident I'll more fully type these out when I get to porting over the Curation pages.